packages/instrumentation/package.json

```diff
@@ -22,9 +22,9 @@
     "typescript": "5.3.3"
   },
   "dependencies": {
-    "@opentelemetry/api": "1.8.0",
-    "@opentelemetry/instrumentation": "0.49.1",
-    "@opentelemetry/sdk-trace-base": "1.22.0"
+    "@opentelemetry/api": "^1.7",
+    "@opentelemetry/instrumentation": "0.48 || 0.49",
```

Comment:
Note that v0 behaves differently than v1
See online example 
https://jubianchi.github.io/semver-check/#/^0.48/0.49.1
<img width="912" alt="Screenshot 2024-03-06 at 18 09 46" src="https://github.com/prisma/prisma/assets/1328733/1dc9d19f-b6c9-40dd-957d-0bcd3b9a3dad">

https://jubianchi.github.io/semver-check/#/0.48%20||%200.49/0.49.1
<img width="927" alt="Screenshot 2024-03-06 at 18 09 29" src="https://github.com/prisma/prisma/assets/1328733/7b6b76dc-8203-432b-aef2-812a35b83e0a">

Reply:
I would actually make it open too

<img width="1160" alt="image" src="https://github.com/prisma/prisma/assets/7002833/63dcc132-9a2d-444a-8676-1c5a3ebc60a4">


what do you think?

Reply:
I think 0.48 appeared here only because pr is very old

Reply:
I've pushed an update to just make it `^0.49`